tests/test_rewards_manager.py

```diff
@@ -0,0 +1,52 @@
+import os
+import pytest
+import json
+from rewards.classes.RewardsManager import RewardsManager
+from tests.utils import mock_boosts, test_cycle, test_start, test_end
+from unittest import TestCase
+
+os.environ["KUBE"] = "False"
```

Comment:
these need to be moved before the RewardsManager import so get_secret isn't called in the env_config import

Reply:
<img width="557" alt="Screen Shot 2021-10-08 at 1 26 47 PM" src="https://user-images.githubusercontent.com/31353111/136598202-faadecc5-9ccf-4f63-a849-127f25ce0bc2.png">
additionally test failing with